### PR TITLE
Implement a new design

### PR DESCRIPTION
**Pull request by gillstrom**
Also changed name to `Streamline Express` and included @anton-johansson's Klarna Recurring part.

Moved `/se/klarna-recurring/` to `/klarna-recurring/`, but we might want to keep it under `/se/`?

**We need change `product.png`.**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3343 @@
+{
+  "name": "estore-express-page",
```
Comment (anton-johansson) [inline package-lock.json]:
```diff
-  "name": "estore-express-page",
+  "name": "streamline-express-page",
```

I get this as soon as I run `npm install` :)

**Review by anton-johansson — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (gillstrom) [reply package-lock.json]: Oh..

Comment (gillstrom) [reply package-lock.json]: FixeD

**Review by anton-johansson — APPROVED** [review]

**Comment by kevva** [comment]
:+1:

File: source/se/failure.html [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
-			<h1>Något gick fel!</h1>
-			<p>
-				Gå tillbaka till vår beställningssidan och testa igen.
-			</p>
-			<a href="/estore-express/se/">
-				<button class="Button Button--primary">Tillbaka</button>
-			</a>
+			<div class="Result">
+				<h1>Något gick fel!</h1>
+				<p>Gå tillbaka till vår beställningssidan och testa igen.</p>
```
Comment (kevva) [inline source/se/failure.html:23]: > Gå tillbaka till vår beställningssidan och testa igen.

Shouldn't it be `Gå tillbaka till vår beställningssida` or `Gå tillbaka till bestälningssidan`?

File: source/klarna-recurring/index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title>Viskan Streamline Express</title>
+		<meta name="description" content="">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="../index.css">
+	</head>
+	<body>
+		<!--[if lt IE 8]>
```
Comment (kevva) [inline source/klarna-recurring/index.html:12]: Is this even necessary anymore?

File: source/se/success.html [hunk 4]
```diff
@@ -18,8 +18,10 @@
 			</div>
 		</header>
 		<div class="Container">
-			<h1>Tack för din beställning</h1>
-			<p>En orderbekräftelse kommer inom kort komma på epost.</p>
+			<div class="Result">
+				<h1>Tack för din beställning</h1>
+				<p>En orderbekräftelse kommer inom kort komma på epost.</p>
```
Comment (kevva) [inline source/se/success.html:23]: `En orderbekräftelse kommer inom kort skickas till din e-post.`?

File: source/se/failure.html (hunk 2 above)
Comment (anton-johansson) [reply source/se/failure.html:23]: Thanks, fixed in 6b7b67a47538cea2318818c5939df154f76e4056. Also fixed the absolute return URL.

File: source/se/success.html (hunk 4 above)
Comment (anton-johansson) [reply source/se/success.html:23]: Thanks, fixed in 6b7b67a47538cea2318818c5939df154f76e4056.

File: source/klarna-recurring/index.html (hunk 3 above)
Comment (anton-johansson) [reply source/klarna-recurring/index.html:12]: I wouldn't know. :(

Comment (gillstrom) [reply source/klarna-recurring/index.html:12]: Probably not no..

**Pull request merged**